### PR TITLE
added 'Assigned'

### PR DESCRIPTION
**Pull request by anitra-marley**
Hey Robert. Does this look good to you?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,5 +95,12 @@
     <script type="text/javascript">
   $('select').select2();
 </script>
+  <script> 
+    $(document).ready(function(){
+     
+       $("option[disabled='disabled']").append('(Assigned)');
```
Comment (opuscule) [inline index.html]: Can we add a "non-breaking space" between the end of ", NY" and the start of the (Assigned) 

Try a Google search for "html non-breaking space"

**Review by opuscule — COMMENTED** [review]
looking for non breaking space

File: index.html (hunk 1 above)
Comment (anitra-marley) [reply index.html]: Sure. Will research now.

Comment (anitra-marley) [reply index.html]: Found the entity. Will add it to code.

File: index.html [hunk 2]
```diff
@@ -70,7 +70,7 @@
 <option>Saratoga, NY</option>
 <option>Schenectady, NY</option>
 <option>Schoharie, NY</option>
-<option disabled="disabled">Schuyler, NY</option>
+<option disabled="disabled">Schuyler, NY&nbsp;</option>
```
Comment (opuscule) [inline index.html]: Can we try adding the &nbsp; in the "append" on jQuery?

**Review by opuscule — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (anitra-marley) [reply index.html]: Ok. Will work on doing so.

**Review by opuscule — APPROVED** [review]
Nice work!

**Pull request merged**